Dockerfile.with-sidecar

```diff
@@ -0,0 +1,38 @@
+# Build Container
+FROM golang:1.13-buster AS build-env
```

Comment:
Please use golang 1.15 as the PR that updates golang has been merged recently.

https://github.com/topolvm/topolvm/blob/adcd4c9fd4ba26e24d1ee130a0e28646350ce7ab/Dockerfile#L2